common/src/main/java/com/viaversion/viaversion/rewriter/BlockRewriter.java

```diff
@@ -173,6 +173,39 @@ public void register() {
         });
     }
 
+    public void registerLevelChunk(C packetType, Type<Chunk> chunkType, Type<Chunk> newChunkType) {
+        registerLevelChunk(packetType, chunkType, newChunkType, null);
+    }
+
+    public void registerLevelChunk(C packetType, Type<Chunk> chunkType, Type<Chunk> newChunkType, BiConsumer<UserConnection, Chunk> chunkRewriter) {
```

Comment:
nullable annotations

Reply:
Fixed, thanks. also made newChunkType NotNull since it's never actually null in any implementations.